### PR TITLE
use new Manifest class

### PR DESCRIPTION
**Pull request by hutnikau**
requires: https://github.com/oat-sa/generis/pull/764

### CODE REVIEW

File: .Jenkinsfile [hunk 1]
```diff
@@ -13,21 +13,6 @@ pipeline {
                     label : 'Create build build directory',
                     script: 'mkdir -p build'
                 )
-
-                withCredentials([string(credentialsId: 'jenkins_github_token', variable: 'GIT_TOKEN')]) {
```
Comment (Babacooll) [inline .Jenkinsfile:17]: Why removing this ?

File: composer.json [hunk 2]
```diff
@@ -40,14 +49,19 @@
...
+            ]
         }
     },
     "require": {
         "php": "^7.1",
         "phpdocumentor/reflection-docblock": "2.*",
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
         "oat-sa/jig": "~0.1",
+        "oat-sa/composer-npm-bridge": "^0.3.0",
+        "oat-sa/generis": "*",
```
Comment (Babacooll) [inline composer.json:64]: You will change that version before merging right ?

File: .Jenkinsfile (hunk 1 above)
Comment (hutnikau) [reply .Jenkinsfile:17]: We do not need resolve dependencies from manifest anymore (this command was used to give package names from extension id's listed in manifest.php and generate new fake composer.json file to install dependent extensions).

File: composer.json (hunk 2 above)
Comment (hutnikau) [reply composer.json:64]: Yes, it will be changed after https://github.com/oat-sa/generis/pull/764 will be merged and released

**Review by gitromba — APPROVED** [review]
LGTM. Unit tests are passing.

**Pull request merged**